joern-cli/frontends/kotlin2cpg/src/main/scala/io/joern/kotlin2cpg/types/TypeConstants.scala

```diff
@@ -1,19 +1,10 @@
 package io.joern.kotlin2cpg.types
 
 object TypeConstants {
-  val any                               = "ANY"
-  val classLiteralReplacementMethodName = "getClass"
-  val initPrefix                        = io.joern.x2cpg.Defines.ConstructorMethodName
-  val kotlinFunctionXPrefix             = "kotlin.Function"
-  val kotlinSuspendFunctionXPrefix      = "kotlin.coroutines.SuspendFunction"
-  val kotlinAlsoPrefix                  = "kotlin.also"
-  val kotlinApplyPrefix                 = "kotlin.apply"
-  val kotlinUnit                        = "kotlin.Unit"
-  val javaLangBoolean                   = "boolean"
-  val javaLangClass                     = "java.lang.Class"
-  val javaLangObject                    = "java.lang.Object"
-  val javaLangString                    = "java.lang.String"
-  val kotlin                            = "kotlin"
-  val tType                             = "T"
-  val void                              = "void"
+  val any                   = "ANY"
```

Comment:
I know you didn't add these, but while we're refactoring, why not change them to conform to scala naming conventions and use uppercase for these constants?
i.e. e.g. `val Void = "void"` etc.

Reply:
There are tons of such constants everywhere in kotlin2cpg. Will do in a separate PR.